Symfony/CS/Tests/Fixer/Symfony/MethodArgumentDefaultValueFixerTest.php

```diff
@@ -139,12 +139,17 @@ function foo(
                 '<?php function a(&$a, $b) {}',
                 '<?php function a(&$a = null, $b) {}',
             ),
-            array(
-                '<?php function a($a = 1, ...$b) {}',
-            ),
-            array(
-                '<?php function a($a = 1, \SplFileInfo ...$b) {}',
-            ),
         );
+
+        if (defined('T_ELLIPSIS')) {
+            $cases[] = array(
+                '<?php function a($a = 1, \SplFileInfo ...$b) {}',
+            );
+            $cases[] = array(
+                '<?php function a($a = 1, ...$b) {}',
+            );
+        }
+
+        return $cases;
```

Comment:
should be reverted. The fixer has extra handling for php versions without `T_ELLIPSIS`.